pkgs/applications/blockchains/chia/default.nix

```diff
@@ -34,7 +34,9 @@ let chia = python3Packages.buildPythonApplication rec {
     aiofiles
     aiohttp
     aiosqlite
+    anyio
     bitstring
+    boto3
```

Comment:
I am curious, why does Chia need boto3? As far as I know, boto is for provisioning AWS (maybe other clouds too?), and I do not see how this would be a requirement for Chia (which is a blockchain).

Reply:
Good question, it appears like AWS can be used [inside Chia DataLayer,](https://github.com/Chia-Network/chia-blockchain/blob/b90d91d46b2029eea4b672b77ab7e9c413d8062d/chia/data_layer/s3_plugin_service.py) which is a method where you store the root of a Merkle tree on-chain for integrity and share the data through more traditional means.